src/device-state.coffee

```diff
@@ -341,6 +349,11 @@ module.exports = class DeviceState extends EventEmitter
 					@setTarget({
 						local: stateFromFile
 					})
+				.then =>
+					if stateFromFile.pinDevice
+						# multi-app warning!
+						# The following will need to be changed once running multiple applications is possible
+						@pinDevice = { commit: commitToPin, app: appToPin } if commitToPin? and appToPin?
```

Comment:
I think it would make sense to add this `if` to the one just above, it feels a little weird as-is